documentation/project-docs/developer-guide.md

```diff
@@ -65,7 +65,14 @@ Run the following command from the root of the repository to run all the .NET Co
 
 The `dotnet` executable in the artifacts directory can be run directly.
 
-However, it's easier to configure a test environment to run the built `dotnet`.
+However, it's easier to configure a test environment to run the built `dotnet`. This test environment is managed by dogfood. 
+The dogfood script starts a new Powershell with the environment configured to redirect SDK resolution to your build.
+
+From that shell your SDK will be available in:
```

Comment:
it is also available in the command prompt environment where you ran the dogfood script, right?

Reply:
Perhaps, though I didn't see it when looking through the commands. the artifacts build and dogfood are both silent, though I think that is a good thing

Reply:
I just mean that after running the dogfood command you can run things from the command line (example : `dotnet workload update` ) using whatever version of the sdk you have built. Maybe that's obvious to some people but if we are explaining the purpose of dogfood, it's probably worth calling out